rest-service/manager_rest/rest/resources_v2_1/deployment_update.py

```diff
@@ -160,26 +157,24 @@ def _get_and_validate_blueprint_and_inputs(deployment_id, request_json):
         return blueprint, inputs, reinstall_list
 
     @staticmethod
-    def _get_params_and_validate(deployment_id,
-                                 request_json,
-                                 preserve_old_behavior=False):
+    def _parse_args(deployment_id, request_json, from_post=False):
```

Comment:
'from_post' not clear..

Reply:
Changed to `using_post_request`

Reply:
Why not "is_plugin_update" although I don't like this specification

Reply:
This isn't related to plugins update, just whether the deployment has been performed via a PUT request or a POST

Reply:
A little bit strange